cli/command/container/opts.go

```diff
@@ -193,8 +193,8 @@ func addFlags(flags *pflag.FlagSet) *containerOptions {
 	flags.MarkHidden("dns-opt")
 	flags.Var(&copts.dnsSearch, "dns-search", "Set custom DNS search domains")
 	flags.Var(&copts.expose, "expose", "Expose a port or a range of ports")
-	flags.StringVar(&copts.ipv4Address, "ip", "", "Container IPv4 address (e.g. 172.30.100.104)")
-	flags.StringVar(&copts.ipv6Address, "ip6", "", "Container IPv6 address (e.g. 2001:db8::33)")
+	flags.StringVar(&copts.ipv4Address, "ip", "", "IPv4 address (e.g. 172.30.100.104)")
+	flags.StringVar(&copts.ipv6Address, "ip6", "", "IPv6 address (e.g. 2001:db8::33)")
```

Comment:
This was suggested in another PR, but we decided not to do it then _just_ for that change, but now that we're updating these, can you add a comma after `e.g.`? (it's [common to put a comma after](http://english.stackexchange.com/questions/16172/should-i-always-use-a-comma-after-e-g-or-i-e)), so

```
IPv4 address (e.g., 172.30.100.104)
IPv6 address (e.g., 2001:db8::33)
```